src/components/LoanCards/KvClassicLoanCardContainer.vue

```diff
@@ -288,7 +288,12 @@ export default {
 					}, 1000);
 				}
 				this.isAdding = false;
-				this.$emit('add-to-basket', { loanId: this.loanId, name: this.loan?.name, success: true });
+				this.$emit('add-to-basket', {
+					loanId: this.loanId,
+					lendAmount,
```

Comment:
Why was this change needed? We haven't changed the ATB for loan cards for a while.